pypesto/optimize/ess/function_evaluator.py

```diff
@@ -261,3 +261,35 @@ def multiple(self, xs: Sequence[np.ndarray]) -> np.array:
         self.n_eval += len(xs)
         self.n_eval_round += len(xs)
         return res
+
+
+def create_function_evaluator(
+    problem: Problem = None,
+    startpoint_method: StartpointMethod = None,
+    n_procs: int = None,
+    n_threads: int = None,
+):
+    """Create a FunctionEvaluator.
+
+    Based on multiprocessing or multithreading, depending on whether
+    ``n_procs`` (number of processes) or ``n_threads`` (number of threads)
+    is specified. If neither is specified, a single-threaded evaluator is
+    returned.
+    """
+    if n_procs and n_threads:
+        raise ValueError(
+            "Only one of `n_procs` and `n_threads` may be specified."
+        )
+
+    if n_procs:
+        return FunctionEvaluatorMP(
+            problem=problem,
+            startpoint_method=startpoint_method,
+            n_procs=n_procs,
+        )
+
+    return FunctionEvaluatorMT(
+        problem=problem,
+        startpoint_method=startpoint_method,
+        n_threads=n_threads or 1,
+    )
```

Comment:
So the default is multithread with one thread. Just out of curiosity, why not Multi process with 1?

Reply:
I'd expect it to be more or less the same. The mp part would still create a pool of one, the mt part doesn't do any extra.